src/common/index.js

```diff
@@ -0,0 +1,10 @@
+import * as utils from './utils';
```

Comment:
So this "common" module can nearly be subsumed by other, existing modules:

1. PageLoading, if reusable, probably belongs in Paragon.
2. utils are useful for APIs, and probably belong in either frontend-auth (as the place we setup our API clients) or some sort of frontend-api library.
3. As mentioned above, configureUserAccountApiService and fetchUserAccount probably just belong in frontend-auth ultimately.

That means this module doesn't even need to exist!  If this were a more complicated MFE with multiple pages (read: modules), then we might bring this back for code that is shared across them but is otherwise specific to the MFE.